source/Hyperspace-Model-Tests/AbstractCURIETest.class.st

```diff
@@ -2,25 +2,24 @@
 An AbstractCURIETest is a test class for testing the behavior of AbstractCURIE
 "
 Class {
-	#name : 'AbstractCURIETest',
-	#superclass : 'TestCase',
-	#category : 'Hyperspace-Model-Tests',
-	#package : 'Hyperspace-Model-Tests'
+	#name : #AbstractCURIETest,
+	#superclass : #TestCase,
+	#category : 'Hyperspace-Model-Tests'
```

Comment:
These changes are a consequence of not using Pharo 12. The Tonel format includes some changes in Pharo 12, save these changes again in this version to avoid unrelated changes.